src/Widgets/Device.vala

```diff
@@ -150,5 +153,8 @@ public class BluetoothIndicator.Widgets.Device : Gtk.ListBoxRow {
         }
 
         icon_image.icon_name = device.icon == null ? DEFAULT_ICON : device.icon;
+        if (device.alias == "Redmi AirDots_R") {
+            icon_image.icon_name = "audio-headphones";
+        }
```

Comment:
We probably shouldn't be doing this kind of workaround. Since this icon is coming from the Bluez interface, it should be proposed there: http://www.bluez.org/